README.md

```diff
@@ -38,7 +38,7 @@ Every template output will be exported as output.
 
 ## Example
 
-```yml
+```yaml
 on: [push]
 name: AzureLoginSample
```

Comment:
I see there was a miss earlier as well, can you update the name here  "AzureARMSample"

Reply:
@pulkitaggarwl done.

Reply:
[commit](https://github.com/Azure/arm-deploy/pull/34/commits)